packages/react-next/src/components/Slider/Slider.scss

```diff
@@ -0,0 +1,264 @@
+@import '../../common/themeCssVariables';
+@import '../../common/highContrast';
+@import '../../common/focusBorder';
+
+/**
+  * Tokens:
+  * The word "active" in the token refers to the selected section of the slider
+  * The word "inactive" in the token refers to the unselected section of the slider
+  */
+
+@mixin activeSectionHoverStyles() {
+  .activeSection {
+    background-color: var($semanticColorsInputBackgroundChecked);
+    @include high-contrast {
+      background-color: Highlight;
+    }
+  }
+}
+
+@mixin activeSectionActiveStyles {
+  .activeSection {
+    background-color: var($semanticColorsInputBackgroundCheckedHovered);
+    @include high-contrast {
+      background-color: Highlight;
+    }
+  }
+}
+
+// Styles for when the slider is hovered or active
+@mixin inactiveSectionActiveStyles {
+  .inactiveSection {
+    background-color: var($semanticColorsInputPlaceholderBackgroundChecked);
+    @include high-contrast {
+      border-color: Highlight;
+    }
+  }
+}
+
+// Styles for when the slider is hovered or active
+@mixin thumbActiveStyles {
+  .thumb {
+    border: 2px solid var($semanticColorsInputBackgroundCheckedHovered);
+    @include high-contrast {
+      bordercolor: Highlight;
+    }
+  }
+}
+
+@mixin slideBoxctiveZeroTickStyles {
+  .zeroTick {
+    background-color: var($semanticColorsInputPlaceholderBackgroundChecked);
+    @include high-contrast {
+      background-color: Highlight;
+    }
+  }
+}
+
+.root {
+  @include fonts-medium;
+  user-select: none;
+
+  &.vertical {
+    margin-right: 8;
+  }
+}
+
+.titleLabel {
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+  .vertical & {
+    flex-direction: column;
+    height: 100%;
+    text-align: center;
+    margin: 8px 0;
+  }
+}
+
+.slideBox {
+  background: transparent;
+  border: none;
+  flex-grow: 1;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  height: 28px;
+  width: auto;
+  padding: 0 8px; // Make room for thumb at ends of line
+  @include focus-border();
```

Comment:
To keep this in the same order shouldn't you have this include at the top of the class?